backend/services/validationService.js

```diff
@@ -74,13 +74,23 @@ function addUserValidation(body, callback) {
     return true;
 }
 
-function validationBodyProperty(body, propName, callback) {
```

Comment:
This method can be useful in future validations when some of body properties should be strictly defined